docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
     image: fculpo/zenbot:latest
     volumes:
       - ./simulations/:/app/simulations/
+      - ./conf.js:/app/conf.js
```

Comment:
nit: Might want to mount it as read-only as to make sure we only change it from one side

e.g.

`- ./config.js:/app/conf.js:ro`

Reply:
Sure, great idea. I updated the pull request.